### PR TITLE
Povray

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/povray/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+./configure \
+    --prefix="$PREFIX" \
+    --with-boost="$PREFIX" \
+    --with-boost-thread=boost_thread \
+    --without-x \
+    --without-openexr \
+    --without-libsdl \
+    COMPILED_BY="conda"
+
+make
```
Comment (chrisburr) [inline recipes/povray/build.sh:13]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/povray/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{% set name = "povray" %}
+{% set version = "3.7.0" %}
+{% set release = "8" %}
+{% set releaseversion = version + "." + release %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```
Comment (chrisburr) [inline recipes/povray/meta.yaml:8]: Why isn't the `8` in the version number? It looks like it should be `3.7.0.8`?

File: recipes/povray/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - boost-cpp
+
+test:
+  commands:
+    - povray --version
+
+about:
+  home: http://www.povray.org
+  license: AGPL-3.0-only
+  license_family: GPL
```
Comment (chrisburr) [inline recipes/povray/meta.yaml:41]: You can drop this field as it's now added automatically:
```suggestion
```

File: recipes/povray/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - automake
+    - make
+  host:
+    - boost
```
Comment (chrisburr) [inline recipes/povray/meta.yaml:25]: `boost` is `boost-cpp` + the Python bindings so I don't think you need it?
```suggestion
```

File: recipes/povray/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/povray/meta.yaml:8]: I though conda is restricted to 3 digits

Comment (jan-janssen) [reply recipes/povray/meta.yaml:6]: My mistake, it seems to work just fine. Thanks again for your review.

Comment (chrisburr) [reply recipes/povray/meta.yaml:6]: No worries, conda follows PEP 440 which allows any number of digits.

**Pull request merged**